### PR TITLE
add job to check operator-sdk generation for baremetal-operator

### PR DESCRIPTION
**Pull request by dhellmann**
Add a job to run the new generation test for the baremetal-operator
repository using the image and tools added in
https://github.com/metal3-io/baremetal-operator/pull/497

### CODE REVIEW

**Comment by dhellmann** [comment]
We should not merge this until we fix https://github.com/metal3-io/baremetal-operator/issues/434

File: prow/config/config.yaml [hunk 1]
```diff
@@ -216,6 +216,28 @@ presubmits:
           value: "http://localhost:5050/v1/"
         image: registry.hub.docker.com/library/golang:1.13.7
         imagePullPolicy: Always
+  - name: operator-sdk-generate
+    always_run: true
```
[the remark this answers is not in the document]
Comment (dhellmann) [reply prow/config/config.yaml]: Cool, done.

How do we make these jobs show up as "required" to merge?

[the remark this answers is not in the document]
Comment (russellb) [reply prow/config/config.yaml]: good question.  :-)

It looks like we need to configure prow's branch protection feature: https://github.com/kubernetes/test-infra/tree/master/prow/cmd/branchprotector

[the remark this answers is not in the document]
Comment (russellb) [reply prow/config/config.yaml]: I can look into this later today

[the remark this answers is not in the document]
Comment (russellb) [reply prow/config/config.yaml]: PR for this in #79

**Pull request merged**